src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg

```diff
@@ -178,8 +178,8 @@ max_pitch_velocity_hi_alt =0.2
 max_pitch_velocity_lo_alt_breakpoint =20000
 max_pitch_velocity_hi_alt_breakpoint =28000
 max_bank = 30,15
-max_bank_acceleration = 1.5
-max_bank_velocity = 5
+max_bank_acceleration = 1.7

```

Comment:
I think these changes snuck in by accident.

Reply:
if you look they comment all the power as watts in the systems.cfg but list the variable as amps in SDK
![image](https://user-images.githubusercontent.com/73177937/96801855-364ae200-13ce-11eb-83fe-5a5feb28389c.png)

Reply:
this was a second PULL those fixe the AP hunting the NAV or ILS (flying back and forth across your trk)

Reply:
> I think these changes snuck in by accident.

this was a second PULL those fix the AP hunting the NAV or ILS (flying back and forth across your trk)

Reply:
not sure why Git added it here when I pushed it separately this is for a AP change

Reply:
I'm not sure how familiar you are with Git so I apologise if I'm staying things you already know. Pull Requests are by branch, so if you pushes both commits to the same branch then they would appear here.

To make a separate pull request you would need to make another branch, push those changes to that branch, and submit that branch for pull request.

Reply:
Ah I had forgot that, thank you it has been 6 yrs a little rusty :) Should I create that in a new pull and resubmit it for change or can it get commited here this time?

Reply:
Btw I excluded the VS directory so they shouldn't show up next time :)

Reply:
No worries! Yeah, if you wouldn't mind submitting separately, that would be awesome. Normally I wouldn't be super strict about stuff like that in this case, but for flight model adjustments it's good to be able to test that separately as those changes can be finicky.

Reply:
Ok will do.